src/hotspot/share/compiler/oopMap.hpp

```diff
@@ -112,7 +112,7 @@ class OopMapValue: public StackObj {
   void write_on(CompressedWriteStream* stream) {
     stream->write_int(value());
     if(is_callee_saved() || is_derived_oop()) {
-      stream->write_int(checked_cast<int>(content_reg()->value()));
+      stream->write_int(content_reg()->value());
```

Comment:
```suggestion
      stream->write_int(checked_cast<juint>(content_reg()->value()));
```

Reply:
Again here, just removing existing unnecessary cast.  Dealing with any further mismatch is out of scope for this PR.